pkg/plugins/platform/local_test.go

```diff
@@ -121,11 +92,11 @@ func TestLocalStart(t *testing.T) {
 		composeFile := system.FileExists(app.DockerComposeYAMLPath())
 		assert.True(composeFile)
 
-		check, err := ContainerCheck(webContainer, "running")
+		check, err := util.ContainerCheck(webContainer, "running")
```

Comment:
This pattern is all reworked as a for loop in other https://github.com/drud/ddev/pull/193.